README.md

```diff
@@ -79,6 +80,59 @@ vars:
     zendesk_schema: your_schema_name 
 ```
 
```

Comment:
The below was in the source readme but not included here. Pasting in case it should be.
```suggestion

> **Note**: When running the package with a single source connector, the `source_relation` column in each model will be populated with an empty string.

```

Reply:
good catch